lib/openstax/aws/stack.rb

```diff
@@ -1,5 +1,78 @@
+require 'json'
+
 module OpenStax::Aws
   class Stack
+    class Event
+      def initialize(aws_stack_event)
+        @aws_stack_event = aws_stack_event
+      end
+            
+      def status
+        @aws_stack_event.data.resource_status
+      end
+            
+      def reason
+        @aws_stack_event.data.resource_status_reason
+      end
+            
+      def failure?
+        %w(
+          ROLLBACK_COMPLETE
+          ROLLBACK_IN_PROGRESS
+          CREATE_FAILED
+          ROLLBACK_FAILED
+          DELETE_FAILED
+          UPDATE_ROLLBACK_FAILED
+          IMPORT_ROLLBACK_FAILED
+        ).include?(status)
+      end
```

Comment:
FYI all the methods in https://github.com/openstax/aws-ruby/blob/802a06a7f642d9e5c4229763b5e003e445e41e17/lib/openstax/aws/stack.rb#L410-L462 are going to break now :-) Also we want a solution that doesn't duplicate these strings.  What's your proposal?

Reply:
I'll make a constants file

Reply:
or a class? does this mean I have to modify all other methods using methods with hardcoded statuses? :sweat_smile:

Reply:
don't make a constants file :-)